### PR TITLE
Exporters/admin region

### PR DESCRIPTION
**Pull request by tommylees112**
Download the Kenyan administrative regions from https://data.humdata.org

### CODE REVIEW

File: src/exporters/admin_boundaries.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        print(f'Renamed file to: {new_fname.as_posix()}')
+
+
+class KenyaAdminExporter(OCHAExporter):
+    country = 'kenya'
+
+    def export(self) -> None:
+        """Export functionality for the Kenya Admin Boundaries as .shp files
+        """
+        urls = {
```
Comment (gabrieltseng) [inline src/exporters/admin_boundaries.py]: nit: Should these URLs be class attributes?

File: tests/exporters/test_srtm.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import unittest
+import unittest.mock
```
Comment (gabrieltseng) [inline tests/exporters/test_srtm.py]: Is this import necessary?

Comment (tommylees112) [reply tests/exporters/test_srtm.py]: it was on my machine!

File: src/exporters/admin_boundaries.py (hunk 1 above)
Comment (tommylees112) [reply src/exporters/admin_boundaries.py]: so they can be accessed as `self.urls`?

Comment (tommylees112) [reply src/exporters/admin_boundaries.py]: so they can be accessed as `self.urls`?

if you think so! yeah for sure

**Comment by tommylees112** [comment]
@gabrieltseng do you want this data to be stored elsewhere? It's in the raw folder which is only picked up by the preprocessors?

The preprocessors save the outputs to `data/analysis/boundaries` which is in the preprocessor PR which has been merged

File: src/exporters/admin_boundaries.py (hunk 1 above)
Comment (tommylees112) [reply src/exporters/admin_boundaries.py]: okay fixed that!

**Pull request merged**